docs/pages/component/props.mdx

```diff
@@ -772,6 +792,17 @@ source={{
     }
   }}
 ```
+#### viewType
+
+<PlatformsList types={['Android']} />
+
+Allow to explicitly specify view type.
+This flag replace `useSecureView` and `useTextureView` fields.
+There are 3 available values:
+- 'textureView': The video is rendered in a texture view. it allows mapping the view on a texture (usefull for 3D).
```

Comment:
maybe typo..?

Reply:
I don't understand sorry, can you propose a change please ?

Reply:
I was just talking about spelling of `useful`

Reply:
I am learning everyday :) I is fixed, Thank you !